From dfe7e51acbe408b85fcb62bcbde9331a20d23823 Mon Sep 17 00:00:00 2001
From: Richard Nelson <wabz@pidgin.im>
Date: Sun, 3 Feb 2008 02:56:09 +0000
Subject: [PATCH] I always forget something... Update ChangeLog and
 POTFILES.in

--HG--
extra : convert_revision : 7fa15fa3aab45c4cd7f0a0b91626a86c0e6822f2
---
 ChangeLog      |    2 ++
 po/POTFILES.in |    1 +
 2 files changed, 3 insertions(+)

diff --git a/ChangeLog b/ChangeLog
index fbf57dc..31ef35d 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -42,6 +42,8 @@ version 2.4.0 (??/??/????):
 	  buddylist. The current options are 'Group Online/Offline' and 'No
 	  Group'.
 	* Added a log viewer
+	* Added the ability to block/unblock buddies - see the buddy context menu
+	  and the menu for the buddy list.
 
 version 2.3.1 (12/7/2007):
 	http://developer.pidgin.im/query?status=closed&milestone=2.3.1
diff --git a/po/POTFILES.in b/po/POTFILES.in
index b13bbbf..d96f530 100644
--- a/po/POTFILES.in
+++ b/po/POTFILES.in
@@ -7,6 +7,7 @@ finch/gntconn.c
 finch/gntconv.c
 finch/gntdebug.c
 finch/gntft.c
+finch/gntlog.c
 finch/gntnotify.c
 finch/gntplugin.c
 finch/gntpounce.c
-- 
1.7.10.4

